src/components/organisms/RegisterForm/RegisterForm.tsx

```diff
@@ -124,6 +130,19 @@ export const RegisterForm: React.FC<{
     [fields]
   );
 
+  const getFieldTranslate = useCallback(
+    (extra: Record<string, any>[]) => {
+      const currLang: string = i18n.language;
```

Comment:
```suggestion
      const currLang: string = i18n.language;
```

i18n can be undefined